app/assets/stylesheets/refinery/stories.scss

```diff
@@ -0,0 +1,86 @@
+.StoriesPage {
```

Comment:
I like this refactor generally but am not sure it should fit inside "refinery" structure in the app because the stories feature was built outside the structure of refinery (though I did suggest that at some point we might want to refactor it as a refinery extension).

Reply:
:ok_hand: 

I was just trying to follow whatever the current directory structure looked like. I'll change where this file lives.